plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java

```diff
@@ -238,6 +240,20 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
         return new ConnectorTableProperties();
     }
 
+    @Override
+    public void createTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, boolean ignoreExisting)
+    {
+        createTable(tableMetadata);
```

Comment:
I assume that `createTable` would throw if table already exists. Can you determine if reason for an exception is that table was already in bigquery. And then ignore the exception if `boolean ignoreExisting` is `true`?

Reply:
We can determine by BigQueryException (code: 409, status: ALREADY_EXISTS). Let me change.